plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeConnectorSmokeTest.java

```diff
@@ -331,7 +331,7 @@ public void testPathUriDecoding()
         String tableName = "test_uri_table_" + randomNameSuffix();
         registerTableFromResources(tableName, "deltalake/uri", getQueryRunner());
 
-        assertQuery("SELECT * FROM " + tableName, "VALUES ('a=equal', 1), ('a:colon', 2), ('a+plus', 3), ('a space', 4), ('a%percent', 5)");
+        assertQuery("SELECT * FROM " + tableName, "VALUES ('a=equal', 1), ('a:colon', 2), ('a+plus', 3), ('a space', 4), ('a%percent', 5), ('a/forwardslash', 6)");
```

Comment:
Could you add test that also writes value with `/`?

Reply:
Thank you @pajaks  for the suggestion.